src/main/java/org/ibp/api/domain/program/ProgramInfoRequest.java

```diff
@@ -0,0 +1,42 @@
+package org.ibp.api.domain.program;
+
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+@AutoProperty
+public class ProgramInfoRequest {
```

Comment:
This class may not be needed if we send programUUID in the URL and take the user from the context